libs/nest-modules/src/lib/context/oldContext.service.ts

```diff
@@ -1,9 +1,13 @@
+/* eslint-disable @nrwl/nx/enforce-module-boundaries */
 import {Injectable, OnModuleInit} from '@nestjs/common'
-import {Context} from './context'
+
+// nx-ignore-next-line
```

Comment:
import type means no actual circular dependency happens in code, just types. This is a temporary workaround until context has been moved.

nx-ignore-next-line removes it from the dependency graph check, so no circular dependency